examples/platforms/simulation/system.c

```diff
@@ -67,52 +68,68 @@ void otSysInit(int aArgCount, char *aArgVector[])
 {
     char *   endptr;
     uint32_t speedUpFactor = 1;
-    int      argIndex      = 0;
+
+    struct option long_options[] = {
+        {"sleep-to-tx", no_argument, 0, 's'},
```

Comment:
How about define an enumeration to represent the argument return values, refer to https://github.com/openthread/openthread/blob/main/src/posix/main.c#L146.